kf/nfconfig.go

```diff
@@ -942,3 +943,311 @@ func getHostInterfaces() (map[string]bool, error) {
 	}
 	return hostIfaces, nil
 }
+
+func (c *NFConfigs) AddAndStartBPF(bpfProg *models.BPFProgram, ifaceName string, direction string) error {
```

Comment:
"direction" the variable name should be rethought. The way it is being used is not really for direction i.e. ingress or egress. It is more of an point to attach the eBPF program.

Reply:
I think we can do it in separate PR